tron/core/actionrun.py

```diff
@@ -194,6 +194,7 @@ def __init__(
         deploy_group=None,
         retries_remaining=None,
         exit_statuses=None,
+        expected_runtime=None,
```

Comment:
Is expected_runtime needed in actionrun? If it is, you might need to handle it to/from saved state.